azure-pipelines.yml

```diff
@@ -172,7 +172,7 @@ stages:
             curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
             python3.8 get-pip.py && \
             pip3 --version && \
-            pip3 install 'setuptools<58.5' wheel --user && \
+            pip3 install setuptools wheel --user && \
```

Comment:
This file also seems to include unintentional changes (not sure where they came from...)